tools/project.py

```diff
@@ -161,6 +161,12 @@ def main():
                         default=False,
                         help="writes tools/export/README.md")
 
+    parser.add_argument("--build",
+                        type=argparse_filestring_type,
+                        dest="build_dir",
+                        default=None,
+                        help="The source (input) directory")
```

Comment:
Is this flag not for the output directory?

Reply:
That's a copy paste error.